packages/sku/src/types/types.ts

```diff
@@ -208,6 +209,16 @@ export interface SkuConfigBase {
    */
   dangerouslySetTSConfig?: (skuTSConfig: any) => any;
 
+  /**
+   * This function provides a way to modify sku's Vitest configuration.
+   * It should only be used in exceptional circumstances where a solution cannot be achieved by adjusting standard configuration options.
+   *
+   * Before customizing your Vitest configuration, please reach out in [#sku-support](https://seek.enterprise.slack.com/archives/CDL5VP5NU) to discuss your requirements and potential alternative solutions.
+   *
+   * @link https://seek-oss.github.io/sku/#/./docs/configuration?id=dangerouslysetvitestconfig
+   */
+  dangerouslySetVitestConfig?: () => TestUserConfig;
```

Comment:
There isn't much point in this being a function if it's not going to receive the existing config for the user to modify. I'd rather this be consistent with the other dangerous APIs unless there's a good reason for it not to be.

Maybe in the followup where we ditch `@sku-lib/vitest` we pass through the base config (or at least the parts we're ok with users modifying) and let users mutate and return it?